### PR TITLE
Update all references to travis-ci.org to point to .com instead

### LINKED ISSUE #1198: Change travis.yml from linux-ppc64le to separte os and arch tags for travsi-ci.com
Issue: travis-ci.com needs different os/arch defines in the travis.yml

https://blog.travis-ci.com/2019-11-12-multi-cpu-architecture-ibm-power-ibm-z

We currently use the method below, which is now marked legacy:
```
os: linux-ppc64le
```

For the new style, we need to use:
```
os: linux
arch: ppc64le
```

### PR DESCRIPTION
**Pull request by mariusvniekerk**
Closes #1198

### CODE REVIEW

**Comment by isuruf** [comment]
The function `add_project_to_travis` is not needed now right?

**Comment by mariusvniekerk** [comment]
@isuruf I'm not quite sure about that.  Does the github app automatically enable things?
We can certainly try it out.

**Comment by mariusvniekerk** [comment]
cc @jayfurmanek

**Comment by jayfurmanek** [comment]
Cool! Was there a change to the anvil image that was needed to get it to work?

**Comment by mariusvniekerk** [comment]
@jayfurmanek yes there was a change made to the anvil image.  The main issue was that there were files owned by `nobody` which corresponded to uid `4294967294`.  This is outside of the mapping space available in most lxd containers which tends to cap out at `2**16` and leads to the untar issues.

**Comment by isuruf** [comment]
>  Does the github app automatically enable things?

Yes

File: conda_smithy/templates/README.md.tmpl [hunk 1]
```diff
@@ -72,7 +72,7 @@ Current build status
   <tr>
     <td>Travis</td>
     <td>
-      <a href="https://travis-ci.org/{{ github.user_or_org }}/{{ github.repo_name }}">
+      <a href="https://travis-ci.com/{{ github.user_or_org }}/{{ github.repo_name }}">
         <img alt="macOS" src="{{ shield }}travis/{{ github.user_or_org }}/{{ github.repo_name }}/{{ github.branch_name }}.svg?label=macOS">
```
Comment (isuruf) [inline conda_smithy/templates/README.md.tmpl]: Is this shield for .org or .com?

File: conda_smithy/templates/travis.yml.tmpl [hunk 2]
```diff
@@ -31,7 +31,12 @@ matrix:
...
-      os: linux-ppc64le
+      os: linux
+      arch: ppc64le
+    {%- elif data.platform.startswith("linux-aarch64") %}
+    - env: CONFIG={{ data.config_name }} UPLOAD_PACKAGES={{ data.upload }} PLATFORM={{ data.platform }} DOCKER_IMAGE={{ data.config["docker_image"][-1] }}
+      os: linux
+      arch: arm64
     {%- elif data.platform.startswith("linux-64") %}
     - env: CONFIG={{ data.config_name }} UPLOAD_PACKAGES={{ data.upload }} PLATFORM={{ data.platform }} DOCKER_IMAGE={{ data.config["docker_image"][-1] }}
       os: linux
```
Comment (isuruf) [inline conda_smithy/templates/travis.yml.tmpl]: Can you add `arch: amd64` here?

File: versioneer.py [hunk 3]
```diff
@@ -14,8 +14,8 @@
 (https://pypip.in/version/versioneer/badge.svg?style=flat)
 ](https://pypi.python.org/pypi/versioneer/)
 * [![Build Status]
-(https://travis-ci.org/warner/python-versioneer.png?branch=master)
-](https://travis-ci.org/warner/python-versioneer)
+(https://travis-ci.com/warner/python-versioneer.png?branch=master)
+](https://travis-ci.com/warner/python-versioneer)
```
Comment (isuruf) [inline versioneer.py]: Can you revert this?

File: conda_smithy/cli.py [hunk 4]
```diff
@@ -212,13 +212,15 @@ def __call__(self, args):
 
         print("CI Summary for {}/{} (can take ~30s):".format(owner, repo))
         if args.travis:
-            ci_register.add_project_to_travis(owner, repo)
+            # travis-ci.com registration is automatic
```
Comment (isuruf) [inline conda_smithy/cli.py]: Technically this is not true for all orgs. We've enabled it for conda-forge.

Comment (mariusvniekerk) [reply conda_smithy/cli.py]: we've also changed all the code to do it for .com

Comment (mariusvniekerk) [reply conda_smithy/cli.py]: So we no longer have `.org` capability

Comment (isuruf) [reply conda_smithy/cli.py]: What I mean is that `.com` registration is automatic, because we have enabled it for conda-forge.

Comment (mariusvniekerk) [reply conda_smithy/cli.py]: Please push the changes that you want in that case, since I am not clear what you actually want here.

**Comment by isuruf** [comment]
Looks good to me now. Have you tried this on a new repo?

**Comment by mariusvniekerk** [comment]
Not entirely sure how to actually run that test without staged-recipes.

**Comment by scopatz** [comment]
LGTM! We can always cut another release if we need.

**Comment by scopatz** [comment]
Thanks @mariusvniekerk!

**Comment by jakirkham** [comment]
Thanks all! Should we make a `3.7.0` release?

**Comment by isuruf** [comment]
I usually create a new feedstock in my user account and run just the function which I just did and it works.

@scopatz, please don't disregard my comments and merge without giving me even 10 minutes to comment.

**Comment by scopatz** [comment]
sorry I am overwhelmed right now and marius asked me to cut a release

**Pull request merged**